test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -1,26 +1,25 @@
 'use strict';
 /*
  * This test is a regression test for joyent/node#8900.
+ *
+ * Node.js 0.10.34 contains the bug that this test detects.
+ * Node.js 0.10.35 contains the fix.
+ *
+ * Due to the introduction of ES6-isms, comment out `'use strict'` and
+ * `require('../common');` to run the test on Node.js 0.10.x.
```

Comment:
Not strictly necessary, but awfully nice. I'd rather `nvm use 0.10.34` and then test as opposed to undo a fix (and hope that the code is still in basically the same place so that I don't have to do any git archaeology or greps in the code base to find it), recompile, and test.